pkg/dbutil/table.go

```diff
@@ -134,6 +134,8 @@ func columnDefToCol(offset int, colDef *ast.ColumnDef) (*model.ColumnInfo, []*as
 			case ast.ColumnOptionComment:
 				// do nothing
 			case ast.ColumnOptionGenerated:
+				// FIXME: use a default string now to make col.IsGenerated() return true, will use real generated expr string later
+				col.GeneratedExprString = "Generated"
```

Comment:
What do you need to do to get the real value for this field?

Reply:
don't need it now,  can judge this column is generated is ok, maybe we need the real value later